mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -150,6 +164,12 @@ class ConvConverter final
     if (auto attr = op->getAttrOfType<StringAttr>("perf_config"))
       cop->setAttr("perf_config", attr);
 
+    if (elementTy.isInteger(8)) {
+      auto quantAttr = rewriter.getAttr<tosa::ConvOpQuantizationAttr>(
+          /*inputZp =*/0, /*weightZp =*/0);
+      cop->setAttr("quantization_info", quantAttr);
+    }
+
```

Comment:
FYI: I ended up figure out that this attribute is the minimal change I need to make int8 convolution valid IR in TOSA. It required an attribute to tell what zero point for intput and weight is. Since we don't do any quantization related compute in middle of the convolution, add the attribute with hardcoded zero as zero point to comply with TOSA standard.

Reply:
Could we handle things by meaningfully setting this attribute and then generating the `linalg.generic`s in `TosaToRock`? Or would that, for example, only work for uniform quantizations?

Reply:
@krzysz00 Maybe, but only where a `TOSA.rescale` is already provided. That was the intended use case of quantization provided by TOSA. (So that tosa.rescale can decide is parameters according to input/weight quantization factors)

In the meeting with MIGraphX, they explicitly mention that will not give us any information related with intput/weight tensor and only pass us with output's quantized information. So this philosophy goes against TOSA. But since the required semantic was against `TOSA.rescale` anyways, we have little work-around but to match the behavior with what MIGraphX has asked for.

Reply:
Got it, seems reasonable to me

Reply:
Maybe add a comment explaining what and why the workaround is?

Reply:
Done in [8aed021](https://github.com/ROCmSoftwarePlatform/rocMLIR/pull/990/commits/8aed02156d5720249f58330a42c5e7fc8a8c919b)